### DISCUSSION TITLE
Dolby Vision profile can't be read by QuickTime

### DISCUSSION THREAD

**Discussion by iBobbyTS** [quietvoid/dovi_tool#102] [General]
I have a .mov file taken by iPhone 13 Pro with dolby vision and can be played correctly with QuickTime Player. It's at cwd named `input.mov`. 
<img width="394" alt="Screen Shot 2022-01-04 at 10 37 21" src="https://user-images.githubusercontent.com/46138297/148002345-f2812907-cf19-460f-9e7d-a840a57d1c83.png">

The commands I ran are the following: 
```
ffmpeg -i input.mov -f hevc -c copy input.hevc
dovi_tool extract-rpu input.hevc
ffmpeg -i input.mov -c:v libx265 -f hevc output.hevc
dovi_tool inject-rpu --input output.hevc --rpu-in RPU.bin
ffmpeg -f hevc -i injected_output.hevc -c copy -vtag hvc1 output.mov
```
 
`output.mov` can't be rendered as Dolby Vision, only the transfer function is indicating that it's HLG
<img width="394" alt="Screen Shot 2022-01-04 at 11 18 42" src="https://user-images.githubusercontent.com/46138297/148005406-0816d00d-8f88-4bf8-916f-17467cffe893.png">

Is there a way I can compress the video stream and keep the RPU stream unmodified and readable?

**Comment by quietvoid** [comment]
I think you need to use `-strict unofficial` to remux to mov with ffmpeg.
However, you can't mux the raw video for it to work.
Use Dolby's muxer first: https://github.com/DolbyLaboratories/dlb_mp4base

I converted this to a discussion.
